### PR TITLE
drf-haystack 

### PR DESCRIPTION
**Pull request by scopatz**
noarch: python

### CODE REVIEW

**Comment by scopatz** [comment]
This is ready for review, @conda-forge/staged-recipes!

**Comment by ocefpaf** [comment]
Feedstocking is broken at the moment. Let's hold off the merge for a while until we figure out what is going on.

**Comment by scopatz** [comment]
@conda-forge/staged-recipes - can this be merged now, please?

File: recipes/drf-haystack/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - django
+    - django-haystack
+    - djangorestframework
```
Comment (isuruf) [inline recipes/drf-haystack/meta.yaml]: Can you add version constraints from https://github.com/inonit/drf-haystack/blob/master/setup.py#L36-L39

Comment (isuruf) [reply recipes/drf-haystack/meta.yaml]: Also need python-dateutil

Comment (ocefpaf) [reply recipes/drf-haystack/meta.yaml]: Specially the `Django` constraint or this package will be broken with `Django 2.0`.

Comment (ocefpaf) [reply recipes/drf-haystack/meta.yaml]: To make it easier here they are:

```
        "Django>=1.11,<2.1",
        "djangorestframework>=3.7",
        "django-haystack>=2.8",
        "python-dateutil"
```

Comment (scopatz) [reply recipes/drf-haystack/meta.yaml]: I think it allows django 2.0, just not future versions of django

**Comment by scopatz** [comment]
Ok! This should be good for another round of reviews

**Pull request merged**